backend/internal/data/sql/queries/shutter_explorer.sql

```diff
@@ -156,4 +140,27 @@ ORDER BY
 -- name: QueryIncludedTxsInSlot :many
 SELECT tx_hash, EXTRACT(EPOCH FROM created_at)::BIGINT AS included_timestamp  
 FROM decrypted_tx 
-WHERE slot = $1 AND tx_status = 'included';
\ No newline at end of file
+WHERE slot = $1 AND tx_status = 'included';
+
+-- name: QueryFromTransactionDetail :one
+SELECT tx_hash as user_tx_hash, encrypted_tx_hash
+FROM transaction_details 
+WHERE tx_hash = $1 OR encrypted_tx_hash = $1
+ORDER BY submission_time DESC
+LIMIT 1;
+
+-- name: QueryDecryptedTXFromSubmittedEvent :one
```

Comment:
Nitpick: The naming sounds incorrect. We collect query data for the whole shielded tx flow not only the decrypted tx. 
Can you rename to something more accurate please.

Reply:
renamed